test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -12480,6 +12480,33 @@ def forward(self, x):
 
         self.run_test(LerpModel(), torch.rand(5, 4, 3))
 
+    @common_utils.parametrize("input_dtype", [torch.cfloat, torch.float])
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_print_tensor_within_torch_nn_module(self, input_dtype: torch.dtype):
+        class PrintTensorOnMyModel(torch.nn.Module):
+            def forward(self, x):
+                # 'print' has side effect calling 'resolve_conj' and 'resolve_neg'.
+                x_firsts = x[:, 0]
+                print(f"x_firsts: {x_firsts}")
+                # 'tolist' has side effect calling 'resolve_conj' and 'resolve_neg'.
+                # Annotation added to pass torch script.
+                _: List[float] = x.tolist()
+                return x_firsts
+
+        m = PrintTensorOnMyModel().eval()
```

Comment:
```suggestion
```

Reply:
`m` is needed, otherwise the model is not instantiated

Reply:
For simplicity and consistency with other tests, I would just call PrintTensorOnMyModel() in run test (comment above).

Reply:
Each test has its own specificities. It is subjective to expect "consistency" in their implementation. The reasoning for keeping the `eval` was that it came from the bug report

Reply:
Thanks for the context! If a test needs different treatment, I recommend adding a comment on the why to help future readers.

FYI I ran lintrunner locally on the file and got this result

![image](https://user-images.githubusercontent.com/11205048/195405377-b4a78fb8-fad6-47a5-9528-d4cf6f88f169.png)

Just for future reference. This is a rather minor detail we can always fix in some other changes in the future. The trick is to remove the trailing comma and let black add them when it sees the need